java/src/test/java/ai/rapids/cudf/ColumnVectorTest.java

```diff
@@ -4111,6 +4111,7 @@ void testExtractRe() {
 
   @Test
   void testExtractAllRecord() {
+    String pattern = "([ab])(\\d)";
```

Comment:
for my edification, so group index 0 in this case is `([ab])`?

Reply:
0 is a special case in Spark that matches the entire regular expression (all the groups). The group indices are 1-indexed.